queries/perl/injections.scm

```diff
@@ -1 +1,2 @@
-(comments) @comment
+; an injections.scm file for nvim-treesitter
+(pod) @pod
```

Comment:
Can you also add the pod parser (or comment this out till then)?

Reply:
Oh; the POD parser is at https://github.com/tree-sitter-perl/tree-sitter-pod/

Reply:
I'll do this today, I added a release branch there too just now

Reply:
has been added